models/category/src/category-draft/presets/change-history-data/with-no-parent.spec.ts

```diff
@@ -0,0 +1,14 @@
+import type { TCategoryDraft } from '../../../types';
+import withNoParent from './with-no-parent';
+
+describe('Category with no parent', () => {
+  it('should return a category containing no parent value', () => {
+    const category = withNoParent().build<TCategoryDraft>();
+
+    expect(category).toEqual(
+      expect.objectContaining({
+        parent: undefined,
```

Comment:
Do we want to test for the entire object or just the fields we are setting in the preset? I tried to look around but I feel like I found possibly cases of both.